### DISCUSSION TITLE
Release timeline for Capacitor 3.0

### DISCUSSION THREAD

**Discussion by levi-ws** [ionic-team/capacitor#4097] [Q&A]
Congrats on the Beta release for Capacitor 3.0 —

Do you have thoughts/visibility yet as to when it will emerge from Beta and be recommended for use in production?

**Comment by imhoffd** [comment]
Probably a month or two, but no guaranteed timeline.

**Reply by jothipandiyanjp** [reply]
Is the beta version is recommended to use in production?
